ui/pages/v2/HelmReleaseDetail.tsx

```diff
@@ -57,7 +51,10 @@ function HelmReleaseDetail({ className, name }: Props) {
             ]}
           />
         </Info>
-        <PageStatus ok={ok} msg={msg} error={error ? true : false} />
+        <PageStatus
+          conditions={helmRelease?.conditions}
+          error={error ? true : false}
```

Comment:
This will convert the value to a bool:

```suggestion
          error={!!error}
```

I would question the need for this, since `PageStatus` can compute it's own error state from `conditions` right?

Reply:
The error prop is actually from whatever the request is on the page (useListSources, useGetHelmRelease) and not about the source itself - should those be separated?

Reply:
This comment confuses me. I was only pointing out a syntactic improvement around ternary operators.

Reply:
Oh just the second part of your comment about the need for the error prop! No biggie